imap/caldav_alarm.c

```diff
@@ -614,6 +630,12 @@ static int process_alarm_cb(icalcomponent *comp,
                        icaltime_as_ical_string(start), alarmno,
                        summary);
             }
+            else if (!hashset_add(seen_alarms, &alarm_id) && suppress_duplicates) {
+                syslog(LOG_ERR, "suppressing duplicate alarm "
```

Comment:
Do we want to syslog each time we suppress an alarm? What if you have say 1,000,000 suppressions.

Would it be better to count the number of suppressed alarms and log that after?

Reply:
I agree.  Keeping a count and logging at the end is probably the best.

Reply:
Agreed and done.

Reply:
Also demoted the log level from LOG_ERR to LOG_INFO